build-tools/packages/version-tools/src/internalVersionScheme.ts

```diff
@@ -146,7 +150,7 @@ export function toInternalScheme(
 		throw new Error(`Couldn't parse ${version} as a semver.`);
 	}
 
-	if (!allowPrereleases && parsedVersion.prerelease.length !== 0) {
+	if (!allowPrereleases && parsedVersion.prerelease.length > 0) {
```

Comment:
@alexvy86, for reference here is where linter apparently says don't use `.length !== 0`. Perhaps it prefers positive tests (why this is changed) and then does expect `.length` is never negative which causes the other change.

Reply:
Ah that's right, it does prefer positive tests, thus why many `!== undefined` get flipped. That probably drives `> 0` too.